bazel/repository_locations.bzl

```diff
@@ -735,9 +735,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Python rules for Bazel",
         project_desc = "Bazel rules for the Python language",
         project_url = "https://github.com/bazelbuild/rules_python",
-        version = "0.7.0",
-        sha256 = "15f84594af9da06750ceb878abbf129241421e3abbd6e36893041188db67f2fb",
-        release_date = "2022-03-11",
+        version = "b4d4b6d9a5d32a199b510ca3cd3273966ebaf987",
```

Comment:
Can you use the latest release `0.8.1` - https://github.com/bazelbuild/rules_python/releases/tag/0.8.1

Reply:
no - its using my pr/fork atm - it will take a bit more to get this working

Reply:
looks like my pr is about to land - will update this when it is and we can see where its at